config/environments/development.rb

```diff
@@ -107,7 +107,7 @@
   ###################################################################
   # Options for Repository_type are 'svn' and 'memory' for now
   # 'memory' is by design not persistent and only used for testing MarkUs
-  REPOSITORY_TYPE = "svn" # use Subversion as storage backend
+  REPOSITORY_TYPE = "git" # use Subversion as storage backend
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.